ts/Core/Globals.ts

```diff
@@ -182,11 +203,12 @@ var H: typeof Highcharts = {
     SVG_NS: SVG_NS,
     chartCount: 0,
     seriesTypes: {} as Highcharts.SeriesTypesDictionary,
+    isPassiveEvent: checkPassiveEvents(),
```

Comment:
Wouldn't a better name here be `supportsPassiveEvents`? I feel that _is_ is not the right word here, because it's not like Highcharts _is_ a passive event.

Reply:
Absolutely right 👍